src/types/redis_bloom_chain.cc

```diff
@@ -45,17 +45,18 @@ void BloomChain::getBFKeyList(const Slice &ns_key, const BloomChainMetadata &met
 }
 
 rocksdb::Status BloomChain::getBFDataList(const std::vector<std::string> &bf_key_list,
-                                          std::vector<std::string> *bf_data_list) {
+                                          std::vector<rocksdb::Slice> *bf_data_list) {
   LatestSnapShot ss(storage_);
   rocksdb::ReadOptions read_options;
   read_options.snapshot = ss.GetSnapShot();
 
   bf_data_list->reserve(bf_key_list.size());
-  for (const auto &bf_key : bf_key_list) {
-    std::string bf_data;
-    rocksdb::Status s = storage_->Get(read_options, bf_key, &bf_data);
+  for (size_t i = 0; i < bf_key_list.size(); ++i) {
```

Comment:
I wonder if there is a reason to change the range-based for loop to the classic one?

Reply:
I think I accidentally did that for testing purposes. I'll change it back.